src/ggrc/models/hooks/issue_tracker/assessment_integration.py

```diff
@@ -1033,16 +1033,19 @@ def _generate_custom_fields(fields):
         List with "custom fields" for issue tracker
     """
     custom_fields = []
+    due_date_value = None
 
-    if fields.get("due_date"):
-      custom_fields.append(
-          {
-              "name": constants.CustomFields.DUE_DATE,
-              "value": fields["due_date"].strftime("%Y-%m-%d"),
-              "type": "DATE",
-              "display_string": constants.CustomFields.DUE_DATE
-          }
-      )
+    if fields.get('due_date'):
+      due_date_value = fields["due_date"].strftime("%Y-%m-%d")
+
+    custom_fields.append(
+        {
+            "name": constants.CustomFields.DUE_DATE,
+            "value": due_date_value,
```

Comment:
Can we replace it with the inline if-statement?

```suggestion
            "value": fields["due_date"].strftime("%Y-%m-%d") if fields.get('due_date') else None,
```

Reply:
In my opinion, this line will be too long and not so readable.